File: Capstone/Capstone/Models/vecinos.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Capstone.Modelo
{
    [Table("vecinos")]
    public class Vecinos
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Required]
        [StringLength(100)]
        public string? Nombres { get; set; }

        [Required]
        [StringLength(100)]
        public string? ApellidoPaterno { get; set; }

        [Required]
        [StringLength(100)]
        public string? ApellidoMaterno { get; set; }

        [Required]
        public DateTime FechaNacimiento { get; set; }
        [Required]
        public string Rut { get; set; }

        [Required]
        public int Edad { get; set; }

        [Required]
        [StringLength(10)]
        public string? Genero { get; set; }

        [Required]
        [StringLength(200)]
        public string? Direccion { get; set; }

        [Required]
        [StringLength(100)]
        public string? Comuna { get; set; }


        [Required]
        [StringLength(100)]
        public string? Email { get; set; }
        public Vecinos(int id, string? nombres, string? apellidoPaterno, string? apellidoMaterno, DateTime fechaNacimiento, string rut, int edad, string? genero, string? direccion, string? comuna, string? email)
        {
            Id = id;
            Nombres = nombres;
            ApellidoPaterno = apellidoPaterno;
            ApellidoMaterno = apellidoMaterno;
            FechaNacimiento = fechaNacimiento;
            Rut = rut;
            Edad = edad;
            Genero = genero;
            Direccion = direccion;
            Comuna = comuna;
            Email = email;
        }

        public override string ToString()
        {
            return $"{Nombres} {ApellidoPaterno}";
        }
    }
}

File: Proyecto Vecinos Unidos/models/vecinos.py
from flask import Flask, jsonify, request
from flask_sqlalchemy import SQLAlchemy
from datetime import datetime


db = SQLAlchemy()

# Modelo de Vecino
class Vecino(db.Model):
    __tablename__ = 'vecinos'
    id = db.Column(db.Integer, primary_key=True)
    rut = db.Column(db.String(12), nullable=False)
    nombres = db.Column(db.String(100), nullable=False)
    apellido_paterno = db.Column(db.String(100), nullable=False)
    apellido_materno = db.Column(db.String(100), nullable=False)
    fecha_nacimiento = db.Column(db.Date, nullable=False)
    rut = db.Column(db.String(20), unique=True, nullable=False)
    genero = db.Column(db.String(10), nullable=False)
    direccion = db.Column(db.String(200), nullable=False)
    comuna = db.Column(db.String(100), nullable=False)
    email = db.Column(db.String(100), nullable=False)


